src/openforms/registrations/contrib/zgw_apis/service.py

```diff
@@ -142,6 +143,28 @@ def create_csv_document(
     return create_document(name, base64_body, options, get_drc=get_drc)
 
 
+def retrieve_attachment_iotype(
+    submission_attachment: SubmissionFileAttachment, default_iotype: str
+) -> str:
```

Comment:
The function name is confusing, the `retrieve` implies to me that an HTTP call is made, but that's not the case.

Wouldn't it make more sense to make this a method on the `SubmissionFileAttachment` too? It basically should only interface with `openforms.formio.service` and internal data structures.